### ISSUE TITLE
Single record view in SQL query results

### ISSUE THREAD

**Issue by majkinetor** [microsoft/vscode-mssql#1762]
Its cumbersome to use horizontal scroll on tables with many columns (which is also typical scenario).

It would be great if Results pane would contain an option to switch to single record view which shows vertical list of columns and their values, i.e.

|h1|h2|h3|
|---|---|---|
|v1|v2|v3|

is shown as

|h|v|
|---|---|
|h1|v1|
|h2|v2|
|h3|v3|

One could switch to this view via hotkey and uses arrows to select next/previous row.

Of the mainstream tools, dbeaver supports this:

![image](https://user-images.githubusercontent.com/85767/95440511-52c03680-0959-11eb-9237-d6ec416eff1a.png)

**Comment by stahler** [comment]
Yes, a vertical view would be a great addition to this extension.  Another editor that does it is SQuirreL.

**Comment by IONtiv** [comment]
This is very much appreciated. One more small caveat. Please make sure that the column names are also sortable. So, original sequence is the one that is returned by the select statement, and when you sort it, it sort on the column names.

**Comment by croblesm** [comment]
Hi everyone,

Thanks for the feedback and for sharing your thoughts on this idea. We agree that a single-record or vertical results view could improve readability for wide tables. However, this request has been open for some time without significant activity, and implementing such a feature would require substantial UI changes beyond our current roadmap.

We’ll go ahead and close this issue as not planned. If there’s renewed interest or additional use cases in the future, please feel free to open a new issue so we can reconsider it with updated priorities and design context.

Thanks again for the suggestions and for helping us improve the MSSQL extension!
